test/DisableTest.php

```diff
@@ -30,15 +33,15 @@ class DisableTest extends TestCase
 
     public function setUp()
     {
-        $this->projectDir = vfsStream::setup('project', null, [
+        $this->projectDir = vfsStream::setup($this->projectDirName, null, [
```

Comment:
This was meant to be a preliminary action before varying the projectDir, which seems to be the problem.
An empty projectDir causes the problem as far as I understand.
I asked Matthew, how to test against that as the projectDir is hardcoded to "project" in the tests.
I also "cleaned up" the code to make use of the same syntax for vfsStream.
I can revert all the changes  if necessary - still, I don't know how to test against an empty projectDir!